app/src/main/res/layout/save_slide_up_fragment.xml

```diff
@@ -0,0 +1,109 @@
+        app:layout_constraintTop_toBottomOf="@+id/viewPager"
+        app:piv_animationType="swap"
+        app:piv_dynamicCount="true"
+        app:piv_interactiveAnimation="true"
+        app:piv_selectedColor="@color/maroon"
+        app:piv_unselectedColor="@color/greyBackground"
+        app:piv_viewPager="@id/viewPager" />
+
+    <TextView
+        android:id="@+id/tv_restaurant_name"
+        android:layout_width="0dp"
+        android:layout_height="30dp"
+        android:layout_marginStart="8dp"
+        android:layout_marginLeft="8dp"
+        android:layout_marginTop="4dp"
+        android:layout_marginEnd="5dp"
+        android:layout_marginRight="5dp"
+        android:text="Restaurant Name"
+        android:textSize="20dp"
+        app:layout_constraintEnd_toStartOf="@+id/milesAway"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/pageIndicatorView" />
+
+    <RatingBar
+        android:id="@+id/ratingBar"
+        style="@android:style/Widget.DeviceDefault.Light.RatingBar.Small"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:numStars="5"
+        android:layout_marginStart="8dp"
+        android:layout_marginLeft="8dp"
+        android:isIndicator="true"
+        android:theme="@style/RatingBar"
+        app:layout_constraintBottom_toBottomOf="@+id/typeOfFood"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@+id/typeOfFood" />
+
+    <TextView
+        android:id="@+id/milesAway"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="5dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginRight="8dp"
+        android:text="3 miles away"
+        android:textSize="12sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="@+id/tv_restaurant_name" />
+
+    <TextView
+        android:id="@+id/typeOfFood"
+        android:layout_width="0dp"
+        android:layout_height="15dp"
+        android:layout_marginStart="8dp"
+        android:layout_marginLeft="8dp"
+        android:layout_marginTop="5dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginRight="8dp"
+        android:layout_marginBottom="20dp"
+        android:text="$$ ∙ Indian, Vegan, Vegetarian"
+        android:textSize="12sp"
+        app:layout_constraintBottom_toTopOf="@+id/btn_event"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/ratingBar"
+        app:layout_constraintTop_toBottomOf="@+id/tv_restaurant_name" />
+
+    <Button
+        android:id="@+id/btn_event"
+        android:layout_width="0dp"
+        android:layout_height="45dp"
+        android:layout_marginTop="15dp"
+        android:background="@color/maroon"
+        android:fontFamily="@font/vesper_libre_bold"
```

Comment:
totally up to you guys, but i wouldn't use this custom font, it's pretty non-standard android to use custom fonts like this (look at apps like gmail, evernote, etc.). i'd just stick with the default roboto for everything, and only vary the sizes, colors, and boldness.